### PR TITLE
Update to match camelcase urls in Adsense.

### PR DESCRIPTION
**Pull request by Hazlitte**
## Summary

Update to match camelcase urls in Adsense.

Addresses issue #2695 

## Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [X] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/js/modules/adsense/util/status.js [hunk 1]
```diff
@@ -134,8 +134,9 @@ export const determineSiteStatus = ( {
 		return undefined;
 	}
 
+	const lowerSiteURL = siteURL.toLowerCase();
 	const hasSiteURL = urlChannels.some( ( urlChannel ) => {
-		return 0 <= siteURL.indexOf( urlChannel.urlPattern );
+		return 0 <= lowerSiteURL.indexOf( urlChannel.urlPattern );
```
Comment (eugene-manuilov) [inline assets/js/modules/adsense/util/status.js:139]: Let's use lower cased `urlPattern` as well:

```suggestion
		return 0 <= lowerSiteURL.indexOf( urlChannel.urlPattern.toLowerCase() );
```

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]
@Hazlitte in general looks good to me, but there is one more thing to implement here:

> Add test coverage for this new behaviour in determineSiteStatus.

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]
@Hazlitte the following is not implemented yet. Please, update tests as requested in the ticket.

> Add test coverage for this new behaviour in determineSiteStatus.

**Comment by Hazlitte** [comment]
@eugene-manuilov Apologies, I didn't spot the test coverage requirement.  I'll get it done.

**Review by eugene-manuilov — APPROVED** [review]
Awesome! Thanks, @Hazlitte!

**Pull request merged**